subprojects/core/src/main/java/org/gradle/util/Clock.java

```diff
@@ -16,11 +16,13 @@
 
 package org.gradle.util;
 
+import org.gradle.internal.time.DefaultEventTimer;
+
 /**
- * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.Clock`.
+ * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.DefaultEventTimer`.
  */
 @Deprecated
-public class Clock extends org.gradle.internal.time.Clock {
+public class Clock extends DefaultEventTimer {
```

Comment:
I think this is fine: the signature of the type hasn't changed, which should mean that backward compatibility is preserved.